api/python/stats.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Place Statistics
+nav_order: 10
+parent: Python
+grand_parent: API
+---
+
+# Get Statistics for Places.
+
+## `get_stats(dcids, stats_var, obs_dates='latest')`
+
+Returns a time series of statistical values for queried places based on the
+[`StatisticalVariable`](https://browser.datacommons.org/kg?dcid=StatisticalVariable) filtered by the specified dates.
+
+**Arguments**
+
+*   `dcids`: A list of dcids of the
+    [`Place`](https://browser.datacommons.org/kg?dcid=Place) to query for.
+
+*   `stats_var`: The dcid of the
+    [`StatisticalVariable`](https://browser.datacommons.org/kg?dcid=StatisticalVariable).
+
+*   `obs_dates`: The dates of [`Observation`](https://browser.datacommons.org/kg?dcid=Observation)s to include. Can be 'latest', 'all', or an list of dates in 'YYYY-MM-DD' format.
+
+**Returns**
+
+A dictionary mapping each `Place` identified by the given `dcid`
+to its name and the time series associated with the
+`StatisticalVariable` identified by the given `stats_var`
+and filtered by `obs_dates`.
+See example below for more detail about how the returned dictionary is
+structured.
+
+**Raises**
+
+*   `ValueError` - If the payload returned by the Data Commons REST API is
+malformed.
+
+Be sure to initialize the library, and specify the API key. Check the [Python library setup guide](/api/python/) for more details.
+
+## Examples
+
+We would like to get the time series of the [number of males
+at least 25 years old that attended 12th grade but did not receive
+a high school diploma (`dc/0hyp6tkn18vcb`)](https://browser.datacommons.org/kg?dcid=dc/0hyp6tkn18vcb)
+in [Arkansas](https://browser.datacommons.org/kg?dcid=geoId/05)
+and [California](https://browser.datacommons.org/kg?dcid=geoId/06).
+
+```python
+>>> get_stats(["geoId/05", "geoId/06"], "dc/0hyp6tkn18vcb", obs_dates="all")
```

Comment:
Documentation looks good! In other Python documentation examples, `import datacommons as dc` is used. Also, it's probably missing dc.get_stats since you imported as dc.

This is the get_places example:

```
>>> import datacommons as dc
>>> dc.set_api_key(YOUR_API_KEY_HERE)
>>> dc.get_places_in(["geoId/06"], "County")
```